lib/gst/player/gstplayer.c

```diff
@@ -2617,13 +2623,24 @@ gst_player_seek_internal_locked (GstPlayer * self)
   remove_tick_source (self);
   self->is_eos = FALSE;
 
-  ret =
-      gst_element_seek_simple (self->playbin, GST_FORMAT_TIME,
-      GST_SEEK_FLAG_FLUSH, position);
+  flags |= GST_SEEK_FLAG_FLUSH;
+  flags |= GST_SEEK_FLAG_ACCURATE;
```

Comment:
ACCURATE is unnecessarily expensive usually. Did you check if it is really required here?